CommonBike/commonbike-site
# Feature/89 merge location list and map together

**This PR brings a reactive location list to the CommonBike app. You'll see that the location list below the map is updated everytime you move the map, zoom in or zoom out.**

Enhances:

- The `/locations` page does load faster
- Only relevant locations are visible

Possible future improvements:

- Show bikes based on distance to the user

____

HOW TO TEST:

1. Go to `/locations`
2. Move the map -> see the location list change on move/zoom

____

NOTE: This branch is based on the `feature/skopei-lock` branch.

If this feature is tested, we will merge both this branch into `develop`. The full `skopei-lock` branch will be merged as well automatically. Together with @mosbuma (and others) we have to test the new `develop` branch very precise afterwards.
